### PR TITLE
Improve markdown files

### PR DESCRIPTION
**Pull request by ahoppen**
Mostly just removing outdated information, merging files and pointing to the new tools page from https://github.com/apple/swift-org-website/pull/679 with instructions on how to set up editors

### CODE REVIEW

**Review by hamishknight — APPROVED** [review]

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,166 @@
...
+    * **Scope**: An assessment of the impact/importance of the change. For example, is the change a source-breaking language change, etc.
+    * **Issue**: The GitHub Issue link if the change fixes/implements an issue/enhancement.
+    * **Original PR**: Pull Request link from the `main` branch.
+    * **Risk**: What is the (specific) risk to the release for taking this change?
+    * **Testing**: What specific testing has been done or needs to be done to further validate any impact of this change?
+    * **Reviewer**: One or more code owners for the impacted components should review the change. Technical review can be delegated by a code owner or otherwise requested as deemed appropriate or useful.
+    ```
+
+> [!TIP]
+> The PR description can be generated using the [release_branch.md](https://github.com/apple/swift-syntax/blob/main/.github/PULL_REQUEST_TEMPLATE/release_branch.md) [pull request template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates). To use this template when creating a PR, you need to add the query parameter:
```
[the remark this answers is not in the document]
Comment (ahoppen) [reply CONTRIBUTING.md:153]: I was adding it. Just forgot to change the URL. Stupid case-sensitive search 😬

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Pull request merged**